### PR TITLE
Transaction / Mutation Table

### LINKED ISSUE #45: Implement the transaction/mutation table

### PR DESCRIPTION
**Pull request by BHenkemans**
Fixes #45.

### CODE REVIEW

**Comment by CodeNamedRobin** [comment]
Fixes #80

**Comment by CodeNamedRobin** [comment]
@BHenkemans fine mutations are not implemented yet.

**Comment by CodeNamedRobin** [comment]
Right now the way the table works does not fully comply with existing pages. POSInfoView and SingleUserView get their transaction in a different way than what MutationTableComponent wants. This needs to be fixed before merging.

**Comment by CodeNamedRobin** [comment]
Still needs:

- [ ] Pagination
- [ ] Deleting of single transactions
- [ ] (Lower priority) filtering and sorting

File: src/utils/paginationUtil.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// import type { PaginatedFinancialMutationResponse } from "@sudosos/sudosos-client";
```
Comment (JustSamuel) [inline src/utils/paginationUtil.ts]: What happend here

**Review by JustSamuel — CHANGES_REQUESTED** [review]
Added some small fixes and left a small remark.

**Review by JustSamuel — APPROVED** [review]
lgtm

**Pull request merged**